plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/api/reports-taxes.php

```diff
@@ -190,140 +244,26 @@ public function test_get_reports_taxes_param() {
 		$tax_report = reset( $reports );
 
 		$this->assertEquals( 2, $tax_report['tax_rate_id'] );
-		$this->assertEquals( 'TestTax 2', $tax_report['name'] );
-		$this->assertEquals( 8, $tax_report['tax_rate'] );
+		$this->assertEquals( 8.0, $tax_report['tax_rate'] );
 		$this->assertEquals( 'CA', $tax_report['country'] );
 		$this->assertEquals( 'ON', $tax_report['state'] );
-		$this->assertEquals( 0, $tax_report['total_tax'] );
-		$this->assertEquals( 0, $tax_report['order_tax'] );
-		$this->assertEquals( 0, $tax_report['shipping_tax'] );
-		$this->assertEquals( 0, $tax_report['orders_count'] );
+		$this->assertEquals( 8.8, $tax_report['total_tax'] );
+		$this->assertEquals( 8, $tax_report['order_tax'] );
+		$this->assertEquals( 0.8, $tax_report['shipping_tax'] );
+		$this->assertEquals( 1, $tax_report['orders_count'] );
 
 		$tax_report = next( $reports );
 
 		$this->assertEquals( 1, $tax_report['tax_rate_id'] );
-		$this->assertEquals( 'TestTax', $tax_report['name'] );
 		$this->assertEquals( 7, $tax_report['tax_rate'] );
 		$this->assertEquals( 'US', $tax_report['country'] );
-		$this->assertEquals( 'GA', $tax_report['state'] );
+		$this->assertEquals( 'NY', $tax_report['state'] );
```

Comment:
Any reason for this change? I mean, I'm partial to NY 😄, but there doesn't seem to be a need for this change.